<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="studentsTable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>561</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Id </string>
     </property>
     <property name="font">
      <font>
       <family>Uroob</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellido</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>-</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>658</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuARCHIVO">
    <property name="title">
     <string>ARCHIVO</string>
    </property>
    <addaction name="newStudentAction"/>
    <addaction name="exitAction"/>
   </widget>
   <addaction name="menuARCHIVO"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="newStudentAction">
   <property name="text">
    <string>Nuevo Estudiante</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
